Add educational content to style guide (#251)
mattermost/mattermost-handbook: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".

An exception is **Customer Education** content which can be written in first or second person to keep the audience focused and interested.

For example: 

> > While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 

Reviewer: ```suggestion
**While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.** 
```

Author: Why would we not use markdown block quote here?

Reviewer: I'm just keeping it aligned with the other examples in the document. But you're welcome to use Markdown block quotes if that works better. :)

>

> >We are here to [support](https://link-to-support-section) you.

Reviewer: ```suggestion
**We are here to [support](https://link-to-support-section) you.**
```
Additional suggestion: Add this directly after the preceding sentence (i.e., remove the line space).

Author: It is supposed to stand alone to give emphasis to the statement.

And as above why should we not use markdown block quotes when we are quoting?

Reviewer: I've provided a comment above. :)

For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".

An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.

For example: 
> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
>
>We are here to [support](https://link-to-support-section) you.

> as opposed to

Reviewer: ```suggestion
As opposed to:
```

> Deploying the Mattermost Mobile App will take time and iteration.

> 

Reviewer: Some extra line spaces here - possibly best to remove them?

**Voice**

> Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it.

Reviewer: Is this accurate for Customer Education content?

Author: I'm not 100% sure here, but as of now I'd say yes.

For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".

> An exception are educational guides, as first person writing helps them to keep the reader interested in  the subject.

Reviewer: ```suggestion
An exception is Customer Education content which is written in first-person to align more closely with the subject matter.
```
I'm 2/5 on this exact wording but I'm inclined toward either a more substantial reason or no reason at all.

Author: I love your wording! To me keeping the reader engaged is the most important part of a CE guide. If I want to educate someone on a matter by using a text, I first of all have to make sure they read it.

How about a mix like 

```suggestion
An exception is Customer Education content which can be written in first-person to keep the audience focused and interested.
```

?

Reviewer: I understand that. My only change is:
"An exception is Customer Education content which is written in first-person to keep the audience focused and interested."

As an aside, I am wary of stating that first-person keeps the audience focused and interested as first-person is more of a tool used in fiction writing than technical content. That said, my perspective is biased and comes from the opposite end of the spectrum and so is just an opinion/comment (especially as I don't know much about sales/marketing writing). :)

Author: Wouldn't that make it impossible to write educational content from second person?

Reviewer: I didn't realize that was an option, so in that case "can be" would be more appropriate. Perhaps we can say "can be written in first or second person"?

> An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.

Reviewer 2: Propose we include an example. It is not clear to a new reader why using second person doesn't keep the audience focused the same way as using first person.

Author: Here's an example from the *Mobile App Deployment Guide*

>A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
>
>We’re here to support you.

With the current doc guidelines this might sound something like this

> Deploying the Mattermost Mobile App will take time and iteration.

The same content is delivered but the first leaves the reader with the comfort of companionship where the second might stop the reader from trying it at all.

Would this be a suitable example?

Author: @jasonblais from our chat yesterday I refined the example:

> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.
>
>We are here to [support](link-to-support-section) you.

As opposed to

> Deploying the Mattermost Mobile App will take time and iteration.

Reviewer 3: @faase The refined example makes sense to me as an outside reader.  Recommend incorporating it into the piece.
